docs/react/your-first-app/7-live-reload.md

```diff
@@ -1,120 +1,205 @@
+              icon: close,
+              role: 'cancel',
+              handler: () => {
+                // Nothing to do, action sheet is automatically closed
+              },
+            },
+          ]}
+          onDidDismiss={() => setPhotoToDelete(undefined)}
+        ></IonActionSheet>
+      </IonContent>
+    </IonPage>
+  );
+};
 
-```tsx
-<IonActionSheet
-  isOpen={!!photoToDelete}
-  buttons={[
-    {
-      text: 'Delete',
-      role: 'destructive',
-      icon: trash,
-      handler: () => {
-        if (photoToDelete) {
-          deletePhoto(photoToDelete);
-          setPhotoToDelete(undefined);
-        }
-      },
-    },
-    {
-      text: 'Cancel',
-      icon: close,
-      role: 'cancel',
-    },
-  ]}
-  onDidDismiss={() => setPhotoToDelete(undefined)}
-/>
+export default Tab2;
 ```
 
-Above, we added two options: `Delete` that calls `deletePhoto` function (to be added next) and `Cancel`, which when given the role of “cancel” will automatically close the action sheet. It's also important to set the onDidDismiss function and set our photoToDelete back to undefined when the modal goes away. That way, when another image is clicked, the action sheet notices the change in the value of photoToDelete.
-
-Next, we need to implement the deletePhoto method that will come from the `usePhotoGallery` hook. Open the file and paste in the following function in the hook:
+Add a click handler to the `<IonImg>` element. When the app user taps on a photo in our gallery, we’ll display an [Action Sheet](../../api/action-sheet.md) dialog with the option to either delete the selected photo or cancel (close) the dialog.
 
 ```tsx
-const deletePhoto = async (photo: UserPhoto) => {
-  // Remove this photo from the Photos reference data array
-  const newPhotos = photos.filter((p) => p.filepath !== photo.filepath);
-
-  // Update photos array cache by overwriting the existing photo array
-  Preferences.set({ key: PHOTO_STORAGE, value: JSON.stringify(newPhotos) });
-
-  // delete photo file from filesystem
-  const filename = photo.filepath.substring(photo.filepath.lastIndexOf('/') + 1);
-  await Filesystem.deleteFile({
-    path: filename,
-    directory: Directory.Data,
-  });
-  setPhotos(newPhotos);
-};
+<IonGrid>
+  <IonRow>
+    {photos.map((photo) => (
+      <IonCol size="6" key={photo.filepath}>
+        {/* CHANGE: Add a click event listener to each image. */}
+        <IonImg src={photo.webviewPath} onClick={() => setPhotoToDelete(photo)} />
+      </IonCol>
+    ))}
+  </IonRow>
+</IonGrid>
 ```
 
-The selected photo is removed from the Photos array first. Then, we use the Capacitor Preferences API to update the cached version of the Photos array. Finally, we delete the actual photo file itself using the Filesystem API.
-
-Make sure to return the `deletePhoto` function so it is as a part of the hook API that we expose:
+Remember that removing the photo from the `photos` array triggers the `cachePhotos` method for us automatically.
```

Comment:
```suggestion
Remember that removing the photo from the `photos` array triggers the `setPhotos` method for us automatically.
```

Nit: `cachePhotos` isn't a method, I think this means to refer to `setPhotos`, which is called by delete and add new photos